src/StateService/Storage/Keys.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Buffers.Binary;
+
+namespace Neo.Plugins.StateService.Storage
+{
+    public static class Keys
+    {
+        public static byte[] StateRoot(uint index)
+        {
+            byte[] buffer = new byte[sizeof(uint) + 1];
+            buffer[0] = 1;
```

Comment:
```suggestion
            buffer[0] = 0xf1;
```
Let's use `0xf1` otherwise it will conflict with prefixes in core.

Reply:
There is no prefix in core. So it will conflict anyway. That's why we use a separate database.